scripts/ubuntu.sh

```diff
@@ -0,0 +1,223 @@
+# system update
+sudo apt-get -y update
+sudo apt-get install -y --fix-missing ${apt_package_list}
+apt-get clean -y;
+update-alternatives --install /usr/bin/python python /usr/bin/python3 10;
+update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 10;
+pip install --upgrade pip;
+pip3 install ${pip_package_list}
+
+install_dir=${HOME}/opt
+mkdir -p ${install_dir}
+
+# hdf5 std directory
+hdf5_libdir=$HOME/opt/hdf5/$build_hdf5
+
+# need to put libhdf5.so on LD_LIBRARY_PATH (Making sure that LD_LIBRARY_PATH is defined first)
+if [ -z $LD_LIBRARY_PATH ]; then
+  export LD_LIBRARY_PATH="${hdf5_libdir}"
+else
+  export LD_LIBRARY_PATH="${hdf5_libdir}:$LD_LIBRARY_PATH"
+fi
+
+
+############
+### MOAB ###
+############
+# pre-setup
+cd ${install_dir}
+check_repo moab
+mkdir -p moab
+cd moab
+
+# clone and version
+git clone --depth 1 --single-branch -b 5.3.0 https://bitbucket.org/fathomteam/moab moab-repo
+cd moab-repo
+mkdir -p build
+cd build
+
+# cmake, build and install
+cmake ../ -DENABLE_PYMOAB=ON \
+          -DCMAKE_INSTALL_PREFIX=${install_dir}/moab \
+          -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_libdir} \
+          -DBUILD_SHARED_LIBS=ON \
+          -DENABLE_BLASLAPACK=OFF \
+          -DENABLE_FORTRAN=OFF    
+make -j 3 
+cd pymoab
+pip install .
+cd ..
+make install
+cd ..
+rm -rf moab-repo
+
+# Adding MOAB/lib to $LD_LIBRARY_PATH and $LIBRARY_PATH
+export LD_LIBRARY_PATH="${install_dir}/moab/lib:$LD_LIBRARY_PATH"
+
+# Adding pymoab to $PYTHONPATH
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version.split('')[0][0:3])')
+if [ -z $PYTHONPATH ]; then
+  export PYTHONPATH="$install_dir/moab/lib/python${PYTHON_VERSION}/site-packages"
+else
+  export PYTHONPATH="$install_dir/moab/lib/python${PYTHON_VERSION}/site-packages:$PYTHONPATH"
+fi
+
+
+#############
+### DAGMC ###
+#############
+# pre-setup check that the directory we need are in place
+cd ${install_dir}
+check_repo dagmc
+mkdir -p dagmc
+cd dagmc
+
+# clone and version
+git clone --depth 1 --branch stable https://github.com/svalinn/DAGMC.git dagmc-repo
+cd dagmc-repo
+git checkout develop
+mkdir build
+cd build
+
+# cmake, build and install
+cmake .. -DMOAB_DIR=${install_dir}/moab \
+         -DCMAKE_INSTALL_PREFIX=${install_dir}/dagmc \
+         -DBUILD_STATIC_LIBS=OFF \
+         -DBUILD_UWUW=OFF \
+         -DBUILD_TALLY=OFF \
+         -DBUILD_MAKE_WATERTIGHT=OFF \
+         -DBUILD_OVERLAP_CHECK=OFF \
+         -DBUILD_TESTS=OFF 
+make -j 3
+make install
+cd ../..
+rm -rf dagmc-repo
+
+# Adding DAGMC/lib to $LD_LIBRARY_PATH and $LIBRARY_PATH
+export LD_LIBRARY_PATH="${install_dir}/dagmc/lib:$LD_LIBRARY_PATH"
+
+# Adding dagmc/bin to $PATH
+export PATH="${install_dir}/dagmc/bin:$PATH"
+
+####################
+### OpenMC API #####
+####################
+
+cd ${install_dir}
+git clone https://github.com/openmc-dev/openmc.git
+cd openmc
+git checkout tags/v0.13.0
+mkdir bld
+cd bld
+cmake .. -DCMAKE_INSTALL_PREFIX=$HOME/.local
+make
+make install
+cd ..
+pip3 install .
```

Comment:
I think we've aliased `pip` to `pip3` already and used just `pip` above. Let's be consistent